app/services/ccms/attribute_value_generator.rb

```diff
@@ -62,9 +62,10 @@ def means_assessment_capital_contribution(_options)
       0 # TODO: CCMS placeholder
     end
 
-    def lead_proceeding_category(_options)
-      'MAT' # TODO: CCMS placeholder
-    end
+    #  commented out pending check with Stephen, AP-836
+    # def lead_proceeding_category(_options)
+    #   'MAT' # TODO: CCMS placeholder
+    # end
```

Comment:
Why does this need to be commented out? What is the purpose of removing this attribute?

Reply:
This is not used, so it causes the coverage test to fail. There is an inconsistency with the tickets so I was leaving it here until I clarified that issue.

Reply:
This has now been removed in a different PR.